src/routes/index.ts

```diff
@@ -1,16 +1,17 @@
 import { Router } from 'express';
 
-import AuthRouter from './auth/auth.router';
-import HealthRouter from './health/health.router';
-import StorageRouter from './storage/storage.router';
+import authRouter from './auth/auth.router';
+import storageRouter from './storage/storage.router';
 
-const Routes = (): Router => {
+const routes = (): Router => {
```

Comment:
none of these are classes. feel free to disregard, but we should keep consistent naming